src/Confluent.Kafka/Admin/DeleteConsumerGroupOffsetsResult.cs

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+using System.Collections.Generic;
+
+
+namespace Confluent.Kafka.Admin
+{
+    /// <summary>
+    ///     The result of delete consumer group offset request
+    ///     (including error status).
+    /// </summary>
+    public class DeleteConsumerGroupOffsetsResult
+    {
+        /// <summary>
+        ///     Consumer group id.
+        /// </summary>
+        public string Group { get; set; }
+
+        /// <summary>
+        ///     Partitions for which the offsets were reset for.
+        /// </summary>
+        public List<TopicPartitionOffset> Partitions { get; set; }
+
+        /// <summary>
+        ///     Error status.
+        /// </summary>
+        public Error Error { get; set; }
```

Comment:
`internal` (from the perspective of the user, "results" are returned where no error is ever expected, reports are returned when they are). No docstring comment because not public. follow the pattern of the other methods.

Reply:
Updated